### PR TITLE
Fix broken links and update link checking docs

### PR DESCRIPTION
**Pull request by BigLep**
This is done in support of https://github.com/filecoin-project/filecoin-docs/issues/2319

### CODE REVIEW

**Comment by BigLep** [comment]
This is the branch protection rule I want to enable:
https://github.com/filecoin-project/filecoin-docs/settings/rules/2131088
github com_filecoin-project_filecoin-docs_settings_rules_2131088

The key callouts are:
For the main branch...
1. Require PRs
2. Require PRs to have one approval before merging
3. Require comments to be resolved before merging
4. Require linkChecker to pass before merging
5. Block force pushes

I will enable this once it gets approval

**Comment by BigLep** [comment]
@eshon and @lanzafame : I think this is ready for review.  Please look at https://github.com/filecoin-project/filecoin-docs/pull/2327#issuecomment-2397823190 as well.

File: README.md [hunk 1]
```diff
@@ -30,6 +31,10 @@ This repository manages the documentation for the [Filecoin network](https://fil
 
 Want to help out? Pull requests (PRs) are always welcome! If you want to help out but aren't sure where to start, check out the [issues board](https://github.com/filecoin-project/filecoin-docs/issues).
 
+### Link checking
+
+Links are checked using [lychee-action](https://github.com/lycheeverse/lychee-action) as confirugred by [check-external-links.yml](.github/workflows/check-external-links.yml).  Working links are required before merging.  If you have a link that should be excluded from checking use [`.lycheeignore`](https://github.com/lycheeverse/lychee-action?tab=readme-ov-file#excluding-links-from-getting-checked)
```
Comment (lanzafame) [inline README.md:36]: I would add that raw links need backticks and not `<code>` blocks around them to work with the link checker.

**Review by lanzafame — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BigLep) [reply README.md]: @lanzafame : i think `<code>` block works too.  I have updated the README.

**Review by lanzafame — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lanzafame) [reply README.md]: Ahh good to know

**Pull request merged**